pyproject.toml

```diff
@@ -1,110 +1,110 @@
 [project]
 name = "astropy"
-dynamic = [
-    "version"
-]
 description = "Astronomy and astrophysics core library"
 readme = "README.rst"
+keywords = [
+    "ascii",
+    "astronomy",
+    "astrophysics",
+    "coordinate",
+    "cosmology",
+    "fits",
+    "fitting",
+    "modeling",
+    "models",
+    "samp",
+    "science",
+    "space",
+    "table",
+    "units",
+    "wcs",
+]
+license = { text = "BSD-3-Clause" }
 authors = [
     { name = "The Astropy Developers", email = "astropy.team@gmail.com" }
 ]
-license = { text = "BSD-3-Clause" }
 requires-python = ">=3.9"
 classifiers = [
     "Intended Audience :: Science/Research",
     "License :: OSI Approved :: BSD License",
     "Operating System :: OS Independent",
     "Programming Language :: C",
     "Programming Language :: Cython",
-    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
```

Comment:
Will this tool auto-add python3.12?

Reply:
It should according to https://pyproject-fmt.readthedocs.io/en/latest/#calculating-max-supported-python-version

Reply:
Now this is nice